functions-methods.js

```diff
@@ -1,19 +1,67 @@
 // Je gaat functies schrijven die we kunnen hergebruiken om sommige emailadressen te checken. Nu zul je gaan merken hoe handig functies kunnen zijn!
 // Je zult hier methoden van het String Object voor nodig hebben, dus pak de paragraaf op EdHub over het String Object er even bij.
 
+// 1. FUNCTIE
+function congratulate() {
+    console.log("Gefeliciteerd!");
+}
 
+// 2. METHODE, want hij is een property op het birthday object
+const birthday = {
+    congratulate: function () {
+        console.log("Gefeliciteerd!");
+    },
+};
 /* Opdracht  1 */
 // Schrijf een functie genaamd getEmailDomain, die een emailadres verwacht en de domeinnaam teruggeeft. Een domeinnaam is hetgeen dat na het @ in het adres staat
+/*let domain=null;
+function getEmailDomain(emailnaam){
+   domain = emailnaam.substring(emailnaam.indexOf('@') + 1);
+   return domain;
+}
+const getDomain = getEmailDomain("n.eeken@novi-education.nl");
+console.log(getDomain);*/
+const domain = {
+    getEmailDomain: function() {
+        return this.email.substring(this.email.indexOf('@') + 1);
+    }
+}
```

Comment:
Het was niet zo zeer de bedoeling dat je het op deze manier moest doen. Het had ook gewoon zo gemogen
```suggestion
function getEmailDomain(email) {
    return email.substring(email.indexOf('@') + 1);
}
```